examples/generator-demo/index.html

```diff
@@ -32,93 +32,204 @@
-        </block>
-      </statement>
-    </block>
-  </xml>
-
   <script>
+    var toolbox = {
+        "kind":"categoryToolbox",
+        "contents":[
+          {
+            "kind":"category",
+            "name":"Logic",
+            "categorystyle":"logic_category",
+            "contents":[
+              {
+                "kind":"block",
+                "type":"controls_if"
+              },
+              {
+                "kind":"block",
+                "type":"logic_compare"
+              },
+              {
+                "kind":"block",
+                "type":"logic_operation"
+              },
+              {
+                "kind":"block",
+                "type":"logic_negate"
+              },
+              {
+                "kind":"block",
+                "type":"logic_boolean"
+              }
+            ]
+          },
+          {
+            "kind":"category",
+            "name":"Loops",
+            "categorystyle":"loop_category",
+            "contents":[
+              {
+                "kind":"block",
+                "type":"controls_repeat_ext",
+                "inputs":{
+                  "TIMES":{
+                    "block":{
+                      "type":"math_number",
+                      "fields":{
+                        "NUM":10
+                      }
+                    }
+                  }
+                }
+              },
+              {
+                "kind":"block",
+                "type":"controls_whileUntil"
+              }
+            ]
+          },
+          {
+            "kind":"category",
+            "name":"Math",
+            "categorystyle":"math_category",
+            "contents":[
+              {
+                "kind":"block",
+                "type":"math_number",
+                "fields":{
+                  "NUM":123
+                }
+              },
+              {
+                "kind":"block",
+                "type":"math_arithmetic"
+              },
+              {
+                "kind":"block",
+                "type":"math_single"
+              }
+            ]
+          },
+          {
+            "kind":"category",
+            "name":"Text",
+            "categorystyle":"text_category",
+            "contents":[
+              {
+                "kind":"block",
+                "type":"text"
+              },
+              {
+                "kind":"block",
+                "type":"text_length"
+              },
+              {
+                "kind":"block",
+                "type":"text_print"
+              }
+            ]
+          }
+        ]
+      }
+
+    
     var demoWorkspace = Blockly.inject('blocklyDiv',
         {media: 'https://unpkg.com/blockly/media/',
-         toolbox: document.getElementById('toolbox')});
-    Blockly.Xml.domToWorkspace(document.getElementById('startBlocks'),
-                               demoWorkspace);
+         toolbox: toolbox});
+         
+    var flyToolbox = {
```

Comment:
Can you update the name of this to `startBlocks`?

Reply:
Updated!